en/deploy/README.md

```diff
@@ -8,7 +8,16 @@ As you learned, a website has to be located on a server. There are a lot of serv
 
 The other external service we'll be using is [GitHub](http://www.github.com), which is a code hosting service. There are others out there, but almost all programmers have a GitHub account these days, and now so will you!
 
-We'll use GitHub as a stepping stone to transport our code to and from PythonAnywhere.
+We'll use GitHub as a stepping stone to transport our code to and from PythonAnywhere.  Once the initial setup is complete, your workflow will look like:
+
+1. Modify your program on your own computer
+2. Test your program (`python manage.py runserver`)
+3. Save your work in your local change tracking system, called "git" (see below)
+4. Save a copy of your work on a public git server, namely GitHub
+5. Log into the server hosting your web application, PythonAnywhere in this tutorial
```

Comment:
"Log into PythonAnywhere." We don't need to introduce the concept of other hosting services at this point; I think it's better to keep things very step-by-step.